src/blog/articles/2020-appreciate-feedback/readme.md

```diff
@@ -0,0 +1,210 @@
+
+My manager called for an 1-on-1, "Why didn't you fill your self-review?". I
+didn't tell him that I didn't remember, "Do we really need to? I mean I working
+well, you know that. Isn't it a waste of time?". That day I got to know that
+360-feedbacks are precursor to the forthcoming calibration meeting, unknown to
+me, these corporate shenanigans, I thought. The calibration meeting was a
+hidden championship where every employee was compared with every other in same
+role. Hidden because you don't represent yourself, instead your manager
+represents you. "But, you know what I have worked on", it struck me, shit—what
+the hell would he remember about my work when I couldn't. Maybe subconsciously I
+thought he's oracle or something, he would know, I can count on him, go be my
+champion and fetch me a promotion. But, it doesn't work like that and I am lucky
+I got know this very early in my career.
+
+## It takes courage
+
+I started writing all major task I was working or peers who interacted with
+me. It was all one big text note, it had all I needed, what I worked on, what
+they worked on, if found something wrong in their work or their behavior, I
+did keep a note of everything important. It helped me in 1-on-1s to set my
+goals, to steer myself into a kind of position I wanted.
+
+Today when I look back, I feel I didn't do justice to my peers. I didn't tell
+them in time, I waited for the 360-feedback because I had content, I could write
+this time and I did write. But that feedback was useless, it was late by 6
+months, they couldn't do anything then, even if they eagerly wanted to do
+something. I set them up for failure. I did. But what else could I do, I didn't
+know you're supposed to tell people, nobody told me. Nobody asked me for
+feedback, no before the D-day. And I didn't have the courage to tell them
+unasked for. I takes courage to give feedback, even more when unasked for.
+
+## We react badly to negative feedback
+
+If someone points out your mistakes or failure, it's natural that you would
+get in a defensive position. We try to divert, come up with excuses—"I wanted
+to write test cases but you know deadline was so harsh, I could barely complete
+the task itself.", guess what the task is still incomplete and whatever you
+say to in defense of a negative feedback, it's always what it is—a defense,
+an excuse, a diversion.
+
+Acceptance, or a question is a good response. Question, in fact, is the best
+response—"I feel that's true, I think I have underestimated the task, how could
+I have done this better?". Ask question, get more information, get actionable
+feedback. And keep telling your head, she is trying to help me, be rational, be
+reasonable, open to suggestions and don't shut her off, listen, listen, listen
+and ask.
+
+## Not taking personally is a superpower
+
+"That's not a good way to do it, it could turn into some serious performance
+issue".
+
+We attach ourselves to our work so much that tend to loose where do we
```

Comment:
\* lose

Reply:
I think `to loose` is correct here

Reply:
https://www.grammarly.com/blog/loose-lose/